src/routes/docs/products/databases/queries/+page.markdoc

```diff
@@ -15,6 +15,7 @@ Appwrite SDKs provide a `Query` class to help you build queries. The `Query` cla
 | Example                 | Description                                                                                                           |
 |------------------------------------|-----------------------------------------------------------------------------------------------------------------------|
 | `Query.select(["name", "title"])`  | Select which attributes should be returned from a document.                                                            |
+| `Query.and([Query.lessThan(5), Query.greaterThan(10)])`  | Returns document if it matches all of the nested sub-queries in the array passed in. |
 | `Query.or([Query.lessThan(5), Query.greaterThan(10)])`  | Returns document if it matches any of the nested sub-queries in the array passed in. |
```

Comment:
These need an attribute and they should be different for each query

Reply:
Done